### DISCUSSION TITLE
I can't login to my osu lazer account or use any internet-related feature

### DISCUSSION THREAD

**Discussion by john2744** [ppy/osu#14456] [Q&A]
so I just start playing osu lazer and I like it so much but I can't log in to my account or even create a new one or use any internet-related feature and the sad part is I can do all of that on the stable ver. but I don't like the UI and I like the in-game beatmap listing so help me plz and thx in advance 
osu lazer v2021.820.0
[performance.log](https://github.com/ppy/osu/files/7028700/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7028701/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7028702/updater.log)
[database.log](https://github.com/ppy/osu/files/7028703/database.log)
[network.log](https://github.com/ppy/osu/files/7028704/network.log)
and that's all the files u might need here u go
@peppy

**Comment by peppy** [comment]
Please try visiting https://osu.ppy.sh/oauth/token in a browser and screenshot what you see. There's something on your network or PC that's blocking / breaking traffic.

**Comment by john2744** [comment]
![Screenshot (18)](https://user-images.githubusercontent.com/53994952/130390676-1be46b58-e48e-4373-a3b7-22a4b4b4aa1d.png)
here that's what I see
and if there is something I can do to fix this plz let me know 
and thx for the fast reply

**Reply by peppy** [reply]
Do you have any antivirus or internet security software? Have you modified your OS in any way (ie. to increase privacy)?

**Reply by john2744** [reply]
yup I have Malwarebytes antivirus and it includes internet security in it but I didn't modify my OS or the antivirus

**Comment by john2744** [comment]
so can u help me 
sorry if I'm being nosy I'm losing hope here 
I went throw a lot of articles and nothing works
@peppy

**Reply by peppy** [reply]
Uninstall this version and try again another time.
